src/stores/GraphStore.ts

```diff
@@ -56,6 +57,9 @@ export class GraphStore {
 	@observable
 	public interval: IntervalOption = 15;
 
+	@observable
+	public defaultInterval: IntervalOption = 15;
```

Comment:
Just make it readonly property like this.steps and place above. And maybe call it staticInterval?